augmentation/augmentation_impl.hpp

```diff
@@ -38,6 +38,16 @@ void Augmentation::Transform(DatasetType& dataset,
       this->ResizeTransform(dataset, datapointWidth, datapointHeight,
         datapointDepth, augmentations[i]);
     }
+    else if(this->HasHorizontalFlipParam(augmentations[i]))
+    {
+      this->HorizontalFlipTransform(dataset, datapointWidth, datapointHeight,
+        datapointDepth, augmentations[i]);
+    }
+    else if(this->HasVerticalFlipParam(augmentations[i]))
+    {
+      this->VerticalFlipTransform(dataset, datapointWidth, datapointHeight,
+        datapointDepth, augmentations[i]);
+    }
     else
```

Comment:
Here I am adding directly in the `for` loop because I am not able to use `augmentationMap`. I tried to use like -

https://stackoverflow.com/questions/14419202/c-map-of-string-and-member-function-pointer but I am getting error. So if can suggest How should I use it? How should I store the function in the map?

Reply:
Hmm, I will give detailed review in a day or two. There are some more changes that we would need to make.

Reply:
okay

Reply:
Could you also share the exact error. I will take a look. Thanks.